packages/lab/src/overlay/OverlayPanel.tsx

```diff
@@ -8,8 +8,12 @@ import { useComponentCssInjection } from "@salt-ds/styles";
 import overlayPanelCss from "./OverlayPanel.css";
 
 const withBaseName = makePrefixer("saltOverlayPanel");
-
-export interface OverlayPanelProps extends ComponentPropsWithoutRef<"div"> {}
+export interface OverlayPanelProps extends HTMLAttributes<HTMLDivElement> {
+  /**
+   * The content of Overlay Panel
+   */
+  children?: ReactNode;
```

Comment:
changed to this structure so that the children props come through in the props table for the site docs

Reply:
Was the change from ComponentPropsWithoutRef neccessary?

Reply:
updated thanks